apps/src/gamelab/GameLab.js

```diff
@@ -73,7 +73,7 @@ var GameLab = function () {
   this.consoleLogger_ = new JsInterpreterLogger(window.console);
 
   /** @type {JsDebuggerUi} */
-  this.debugger_ = new JsDebuggerUi(this.runButtonClick.bind(this));
+  this.debugger_ = null;
```

Comment:
Sounds good.  Maybe later we can make them both do the sane thing at the same time :)